gnovm/memfile.go

```diff
@@ -87,6 +89,25 @@ func (mempkg *MemPackage) Validate() error {
 		prev = file.Name
 	}
 
+	pIndex := strings.Index(mempkg.Path, "/p/")
+	if pIndex > 0 && strings.Count(mempkg.Path[:pIndex], "/") == 0 {
+		for _, file := range mempkg.Files {
+			fset := token.NewFileSet()
```

Comment:
I think we should not spend resources trying to parse non-gno files

```suggestion
		for _, file := range mempkg.Files {
			if !strings.HasSuffix(file.Name, ".gno") {
				continue
			}
			fset := token.NewFileSet()
```